src/mbtrn/strptime.c

```diff
@@ -0,0 +1,252 @@
+// https://stackoverflow.com/questions/667250/strptime-in-windows 
```

Comment:
Add the license

Reply:
I would, but there is no license in that post of SO.